src/TextUI/Command/Commands/ListTestsAsTextCommand.php

```diff
@@ -35,8 +35,15 @@ public function execute(): Result
 
         $buffer .= 'Available test(s):' . PHP_EOL;
 
+        $configuration = Registry::get();
+        $configuredGroups = $configuration->groups();
+
         foreach (new RecursiveIteratorIterator($this->suite) as $test) {
             if ($test instanceof TestCase) {
+                if (!$this->testContainsConfiguredGroups($test, $configuredGroups)) {
```

Comment:
should a `PhptTestCase` be skipped when groups are passed via CLI flags, since a `PhptTestCase` cannot define a group (AFAICT) ?

I would skip them - do you agree?